components/tools/OmeroPy/src/omero/plugins/web.py

```diff
@@ -82,15 +84,18 @@ def _configure(self, parser):
 
         config = parser.add(
             sub, self.config,
-            "Output a config template for server"
-            " ('nginx' or 'apache' for the moment)")
-        config.add_argument("type", choices=("nginx", "apache"))
+            "Output a config template for web server\n"
+            "  nginx: Nginx system configuration for inclusion\n"
+            "  nginx-development: Standalone user-run Nginx server\n"
+            "  apache: Apache 2.2 with mod_fastcgi\n"
+            "  apache-fcgi: Apache 2.4+ with mod_proxy_fcgi\n")
+        config.add_argument("type", choices=(
+            "nginx", "nginx-development", "apache", "apache-fcgi"))
         config.add_argument(
             "--http", type=int,
             help="HTTP port for web server (not fastcgi)")
         config.add_argument(
-            "--system", action="store_true",
```

Comment:
@joshmoore I took out all the flags because they were inconsistent- they only worked for nginx and not apache. I suspect few people would script the omero config web-server config with every upgrade since it requires a mix of commands being run by the `omero` and `root` system users (nginx conf directory shouldn't be writeable by `omero`). If you think people are likely to have it scripted I could add `--system` as a noop.